jupyterlab_gcedetails/src/data.ts

```diff
@@ -105,6 +111,110 @@ export const REFRESHABLE_MAPPED_ATTRIBUTES = [
 
 MAPPED_ATTRIBUTES.push(...REFRESHABLE_MAPPED_ATTRIBUTES);
 
+/**
+ * AI Platform Accelerator types.
+ * https://cloud.google.com/ai-platform/training/docs/using-gpus#compute-engine-machine-types-with-gpu
+ */
+export const ACCELERATOR_TYPES: Option[] = [
+  { value: '', text: 'None' },
+  { value: 'NVIDIA_TESLA_K80', text: 'NVIDIA Tesla K80' },
+  { value: 'NVIDIA_TESLA_P4', text: 'NVIDIA Tesla P4' },
+  { value: 'NVIDIA_TESLA_P100', text: 'NVIDIA Tesla P100' },
+  { value: 'NVIDIA_TESLA_T4', text: 'NVIDIA Tesla T4' },
+  { value: 'NVIDIA_TESLA_V100', text: 'NVIDIA Tesla V100' },
+];
+
+/**
+ * AI Platform Accelerator counts.
+ * https://cloud.google.com/ai-platform/training/docs/using-gpus
+ */
+export const ACCELERATOR_COUNTS_1_2_4_8: Option[] = [
+  { value: '1', text: '1' },
+  { value: '2', text: '2' },
+  { value: '4', text: '4' },
+  { value: '8', text: '8' },
+];
+
+/**
+ * AI Platform Machine types.
+ * https://cloud.google.com/ai-platform/training/docs/machine-types#compare-machine-types
+ */
+export const MASTER_TYPES: Option[] = [
+  { value: 'n1-standard-4', text: '4 CPUs, 15 GB RAM' },
+  { value: 'n1-standard-8', text: '8 CPUs, 30 GB RAM' },
+  { value: 'n1-standard-16', text: '16 CPUs, 60 GB RAM' },
+  { value: 'n1-standard-32', text: '32 CPUs, 120 GB RAM' },
+  { value: 'n1-standard-64', text: '64 CPUs, 240 GB RAM' },
+  { value: 'n1-standard-96', text: '96 CPUs, 360 GB RAM' },
+
+  { value: 'n1-highmem-2', text: '4 CPUs, 26 GB RAM' },
+  { value: 'n1-highmem-4', text: '4 CPUs, 26 GB RAM' },
+  { value: 'n1-highmem-8', text: '8 CPUs, 52 GB RAM' },
+  { value: 'n1-highmem-16', text: '16 CPUs, 104 GB RAM' },
+  { value: 'n1-highmem-32', text: '32 CPUs, 208 GB RAM' },
+  { value: 'n1-highmem-64', text: '64 CPUs, 416 GB RAM' },
+  { value: 'n1-highmem-96', text: '96 CPUs, 624 GB RAM' },
+
+  { value: 'n1-highcpu-16', text: '16 CPUs, 14.4 GB RAM' },
+  { value: 'n1-highcpu-32', text: '32 CPUs, 28.8 GB RAM' },
+  { value: 'n1-highcpu-64', text: '64 CPUs, 57.6 GB RAM' },
+  { value: 'n1-highcpu-96', text: '96 CPUs, 86.4 GB RAM' },
+];
+
+export interface HardwareConfiguration {
+  cpu: number;
+  memory: number;
+}
+
+/* CPU to Memory mappings for the Compute Engine machine types */
+interface CEMachineType {
```

Comment:
Can we just name this MachineType?

Reply:
MachineType is being used in the file to define the machine the user currently has:
```
interface MachineType {
  name: string;
  description: string;
}

interface Instance {
  attributes: {
    framework: string;
    title: string;
    version: string;
  };
  cpuPlatform: string;
  id: number;
  image: string;
  machineType: MachineType;
  name: string;
  zone: string;
}
```
Thoughts on changing CEMachineType to MachineTypeConfigurations instead?

Reply:
Yes, let's go with that.